conf/broker.conf

```diff
@@ -61,6 +61,12 @@ numIOThreads=
 # Number of threads to use for HTTP requests processing. Default is set to 2 * Runtime.getRuntime().availableProcessors()
 numHttpServerThreads=
 
+# Number of thread pool size to use for pulsar broker service.
+# The executor in thread pool will do basic broker operation like load/unload bundle, update managedLedgerConfig,
+# update topic/subscription/replicator message dispatch rate, do leader election etc.
+# Default is 20
+numExecutorThreadPoolSize=
```

Comment:
I would be better to default to something like `Runtime.getRuntime().availableProcessors()`

Reply:
ok, it's better to change it to Runtime.getRuntime().availableProcessors(). it will update the pr.